t/plugin/zipkin2.t

```diff
@@ -126,8 +126,9 @@ b3: 80f198ee56343ba864fe8b2a57d3eff7-e457b5a2e4d86bd1-0-05e3ac9a4f6e3b90
 --- response_headers
 x-b3-sampled: 0
 x-b3-traceid: 80f198ee56343ba864fe8b2a57d3eff7
-x-b3-parentspanid: 05e3ac9a4f6e3b90
-x-b3-spanid: e457b5a2e4d86bd1
+x-b3-parentspanid: e457b5a2e4d86bd1
+--- raw_response_headers_like:
```

Comment:
It should be:
```
--- raw_response_headers_like
.*x-b3-spanid:.*
```

Reply:
@spacewander  Sorry, I'm not familiar with this testing framework. I need some help. 
The expected test result is that the `x-b3-spanid`  header exists and is not empty.  Because `x-b3-spanid` header's value is a random value.

May be look like this  ?
```
--- raw_response_headers_like: x-b3-spanid: .*
```

Reply:
LGTM